core/src/main/java/org/apache/gravitino/listener/FilesetEventDispatcher.java

```diff
@@ -32,16 +32,22 @@
 import org.apache.gravitino.file.FilesetChange;
 import org.apache.gravitino.listener.api.event.AlterFilesetEvent;
 import org.apache.gravitino.listener.api.event.AlterFilesetFailureEvent;
+import org.apache.gravitino.listener.api.event.AlterFilesetPreEvent;
 import org.apache.gravitino.listener.api.event.CreateFilesetEvent;
 import org.apache.gravitino.listener.api.event.CreateFilesetFailureEvent;
+import org.apache.gravitino.listener.api.event.CreateFilesetPreEvent;
 import org.apache.gravitino.listener.api.event.DropFilesetEvent;
 import org.apache.gravitino.listener.api.event.DropFilesetFailureEvent;
+import org.apache.gravitino.listener.api.event.DropFilesetPreEvent;
 import org.apache.gravitino.listener.api.event.GetFileLocationEvent;
 import org.apache.gravitino.listener.api.event.GetFileLocationFailureEvent;
+import org.apache.gravitino.listener.api.event.GetFileLocationPreEvent;
 import org.apache.gravitino.listener.api.event.ListFilesetEvent;
 import org.apache.gravitino.listener.api.event.ListFilesetFailureEvent;
+import org.apache.gravitino.listener.api.event.ListFilesetPreEvent;
 import org.apache.gravitino.listener.api.event.LoadFilesetEvent;
 import org.apache.gravitino.listener.api.event.LoadFilesetFailureEvent;
+import org.apache.gravitino.listener.api.event.LoadFilesetPreEvent;
```

Comment:
Is `import org.apache.gravitino.listener.api.event.*` anti-pattern thus discouraged?

Reply:
yes, import xx.* is not preferred